packages/devtools_app/lib/src/profiler/cpu_profile_model.dart

```diff
@@ -379,6 +446,15 @@ class CpuStackFrame extends TreeNode<CpuStackFrame>
     }
   }
 
+  Map<String, dynamic> get json => {
```

Comment:
nit: rename to toJson() for consistency and because this generates a new object each time.

Reply:
done